vagrant/setuplib.sh

```diff
@@ -214,20 +247,58 @@ function setuplib::python3 {
     yum install -q -y python36 python36-devel
 }
 
-function setuplib::mod_wsgi {
-    echo "========= Installing Mod WSGI for Python 3 ==========="
-    yum install -q -y centos-release-scl
-    yum install -q -y rh-python36-mod_wsgi
-    cp /opt/rh/httpd24/root/usr/lib64/httpd/modules/mod_rh-python36-wsgi.so /lib64/httpd/modules
-    cp /opt/rh/httpd24/root/etc/httpd/conf.modules.d/10-rh-python36-wsgi.conf /etc/httpd/conf.modules.d
-    systemctl restart httpd
+
+# Prepare a node to be a Kive worker 
+function setuplib::kive_worker {
+    echo "========= Installing Kive worker components ==========="
+    usermod -a -G kive vagrant
+    yum groupinstall -q -y 'Development Tools'
+    yum install -q -y python-devel libsqlite3x-devel words lsof graphviz graphviz-devel
+
+    python3 -m venv /opt/venv_kive
+    . /opt/venv_kive/bin/activate
+    python -m pip install --upgrade pip
+    pushd /usr/local/share/Kive/
+    python -m pip install -r requirements-dev.txt
+    popd
 }
 
-function setuplib::kive {
-    echo "========= Installing Kive ==========="
+# Set a node up to host the Kive web application
+function setuplib::kive_head {
+    echo "========= Installing Kive application ==========="
     python3 -m venv /opt/venv_kive
     cd /usr/local/share/Kive/vagrant
     ./kive_setup.bash requirements-dev.txt
```

Comment:
Same question here about the requirements file, although the dev version might actually be useful for doing development work on the head node. However, I suggest we install `requirements.txt` by default, and then manually install the rest if needed.